easybuild/easyblocks/o/openmpi.py

```diff
@@ -140,6 +140,31 @@ def test_step(self):
 
         super(EB_OpenMPI, self).test_step()
 
+    def make_module_req_guess(self):
+        """Special case for OpenMPI-CUDA"""
```

Comment:
@akesandgren This docstring should be fine-tuned a bit.

Although the current implementation only does specific stuff for the `OpenMPI-CUDA` case, that may not stay list this going forward.

The method docstring should just say something like "`Tweaks to path guesses specific to OpenMPI.`", and a comment should be added right above the `if` to explain why the `PATH` entry is removed, and why the `OMPI_MCA_mca_component_path` is added.

Similar remark for `make_module_extra` below.
That seems like a duplicate of what's being done here in `make_module_req_guess`, which are both needed?!